tests/rest/client/test_sync.py

```diff
@@ -417,7 +419,23 @@ def test_hidden_read_receipts(self):
         # Test that the first user can't see the other user's hidden read receipt
         self.assertEqual(self._get_read_receipt(), None)
 
-    def test_read_receipt_with_empty_body(self):
+    @parameterized.expand(
+        [
+            (
+                "agent1",
+                "Element/1.2.2 (Linux; U; Android 9; MatrixAndroidSDK_X 0.0.1)",
+                200,
+            ),
+            ("agent2", "Element/1.2.2 (Linux; Android 6)", 200),
+            ("agent3", "SchildiChat/1.3.6 (Android 11)", 200),
```

Comment:
I was under the impression that 1.3+ should fail
```suggestion
            ("agent3", "SchildiChat/1.3.6 (Android 11)", 400),
```

(maybe add a pass case for SchildiChat too such as
```
            ("agent7", "SchildiChat/1.1.6 (Android 11)", 200),
```
)

Reply:
Ah, I changed that last minute and forgot about the 1.```[012]```. restriction, so this should indeed fail.